kujira/assetlist.json

```diff
@@ -1,1160 +1,2264 @@
+                }
+            ]
         },
         {
-          "denom": "avax",
-          "exponent": 18
-        }
-      ],
-      "type_asset": "ics20",
-      "base": "ibc/004EBF085BBED1029326D56BE8A2E67C08CECE670A94AC1947DF413EF5130EB2",
-      "name": "Axelar Wrapped AVAX",
-      "display": "avax",
-      "symbol": "wAVAX",
-      "traces": [
-        {
-          "type": "ibc",
-          "counterparty": {
-            "chain_name": "axelar",
-            "base_denom": "wawax-wei",
-            "channel_id": "channel-14"
-          },
-          "chain": {
-            "channel_id": "channel-9",
-            "path": "transfer/channel-9/wavax-wei"
-          }
-        }
-      ],
-      "images": [
+            "description": "LP Bow: DOT.axl-MNTA",
+            "denom_units": [
+                {
+                    "denom": "factory/kujira10qyzrqaza0f593cmnl4sc3p3nhqqakn9j56d899u723pznxpn8us6cypuw/ulp",
+                    "exponent": 0
+                },
+                {
+                    "denom": "lp-dot-mnta",
+                    "exponent": 6
+                }
+            ],
+            "base": "factory/kujira10qyzrqaza0f593cmnl4sc3p3nhqqakn9j56d899u723pznxpn8us6cypuw/ulp",
+            "name": "LP Bow: DOT.axl-MNTA",
+            "display": "lp-dot-mnta",
+            "symbol": "LP DOT.axl-MNTA",
+            "logo_URIs": {
+                "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
+            },
+            "images": [
+                {
+                    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
+                }
+            ]
+        },
         {
-          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/avalanche/images/wavax.svg"
-        }
-      ],
-      "logo_URIs": {
-        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/avalanche/images/wavax.svg"
-      }
-    },
-    {
-      "denom_units": [
+            "description": "LP Bow: SHD-MNTA",
+            "denom_units": [
+                {
+                    "denom": "factory/kujira12tpc335t72qtrjh64ccdydzsr9qtswkqn72cyvfj4d5hmtgynphsv4xjtu/ulp",
+                    "exponent": 0
+                },
+                {
+                    "denom": "lp-shd-mnta",
+                    "exponent": 6
+                }
+            ],
+            "base": "factory/kujira12tpc335t72qtrjh64ccdydzsr9qtswkqn72cyvfj4d5hmtgynphsv4xjtu/ulp",
+            "name": "LP Bow: SHD-MNTA",
+            "display": "lp-shd-mnta",
+            "symbol": "LP SHD-MNTA",
+            "logo_URIs": {
+                "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
+            },
+            "images": [
+                {
+                    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
```

Comment:
For cases where you use bow.svg, can we also define bow.png as a new image object? i.e., 
```
"images": [
  {
    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
  },
  {
    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.png"
  }
]
```

Reply:
Or else remove bow.png because it is unused

Reply:
Yes, bow.png was removed from all items on the last commit.

Reply:
It was removed from the code but the file is still there under images. I'm saying we should make use of it (by specifying it as an alternative, or else remove the file from /images/)

Reply:
Ah yeah! My bad. Removed the file. Cheers